main.sh

```diff
@@ -30,6 +30,53 @@ warn() {
 info() {
     echo "info: $*"
 }
+cargo_binstall() {
+    tool="$1"
+    version="$2"
+
```

Comment:
It would be nice to have info message like "install-action does not support this tool, fallback to cargo-binstall" here.